libs/ui/src/lib/task-selector/task-selector.component.spec.ts

```diff
@@ -0,0 +1,313 @@
+            "collection": "@schematics/angular",
+            "__typename": "Schematic"
+          }
+        },
+        {
+          "taskName": "enum",
+          "task": {
+            "name": "enum",
+            "description": "Create an enumeration.",
+            "collection": "@schematics/angular",
+            "__typename": "Schematic"
+          }
+        },
+        {
+          "taskName": "guard",
+          "task": {
+            "name": "guard",
+            "description": "Create a guard.",
+            "collection": "@schematics/angular",
+            "__typename": "Schematic"
+          }
+        },
+        {
+          "taskName": "interface",
+          "task": {
+            "name": "interface",
+            "description": "Create an interface.",
+            "collection": "@schematics/angular",
+            "__typename": "Schematic"
+          }
+        },
+        {
+          "taskName": "module",
+          "task": {
+            "name": "module",
+            "description": "Create an Angular module.",
+            "collection": "@schematics/angular",
+            "__typename": "Schematic"
+          }
+        },
+        {
+          "taskName": "pipe",
+          "task": {
+            "name": "pipe",
+            "description": "Create an Angular pipe.",
+            "collection": "@schematics/angular",
+            "__typename": "Schematic"
+          }
+        },
+        {
+          "taskName": "service",
+          "task": {
+            "name": "service",
+            "description": "Create an Angular service.",
+            "collection": "@schematics/angular",
+            "__typename": "Schematic"
+          }
+        },
+        {
+          "taskName": "universal",
+          "task": {
+            "name": "universal",
+            "description": "Create an Angular universal app.",
+            "collection": "@schematics/angular",
+            "__typename": "Schematic"
+          }
+        },
+        {
+          "taskName": "appShell",
+          "task": {
+            "name": "appShell",
+            "description": "Create an app shell.",
+            "collection": "@schematics/angular",
+            "__typename": "Schematic"
+          }
+        },
+        {
+          "taskName": "library",
+          "task": {
+            "name": "library",
+            "description": "Generate a library project for Angular.",
+            "collection": "@schematics/angular",
+            "__typename": "Schematic"
+          }
+        }
+      ]
+    },
+    {
+      "collectionName": "@schematics/update",
+      "tasks": [
+        {
+          "taskName": "update",
+          "task": {
+            "name": "update",
+            "description": "Update one or multiple packages to versions, updating peer dependencies along the way.",
+            "collection": "@schematics/update",
+            "__typename": "Schematic"
+          }
+        }
+      ]
+    }
+  ]
+});
+
+interface Task {
```

Comment:
I believe there's an existing interface you can import

Reply:
couldnt find the exact type in the repo. Which type did you have in mind?

Reply:
Just my imagination actually. Sorry about that

Reply:
haha. no worries :)